tests/test-systray.c

```diff
@@ -0,0 +1,98 @@
+/* Code  https://stackoverflow.com/questions/45392284/freedesktop-xembed-systray-client-code-in-c-xlib */
```

Comment:
Somehow I doubt that this code is licensed as GPLv2...

Reply:
it's `Creative Commons Attribution-ShareAlike 3.0 license`